config/locales/client.en.yml

```diff
@@ -2,6 +2,5 @@ en:
   js:
     login:
       microsoft_office365:
```

Comment:
Is there a reason for not renaming the `microsoft_office365` key?

Reply:
Yes, this key is tied to the 'callback url'. If we change it, everyone would have to update their config with microsoft. I don't think it's worth that pain.